usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/PetCollectionField.java

```diff
@@ -0,0 +1,55 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.Map;
+
+/**
+ * @author Marc Simó
+ */
+public class PetCollectionField {
+    private String key;
+    private Map<String, Object> body;
+
+    public PetCollectionField() { }
+
+    public PetCollectionField(String key, Map<String, Object> body) {
+        this.key = key;
+        this.body = body;
+    }
+
+    public String getKey() {
+        return key;
+    }
+
+    public void setKey(String key) {
+        this.key = key;
+    }
+
+    public Map<String, Object> getBody() {
+        return body;
+    }
+
+    public void setBody(Map<String, Object> body) {
+        this.body = body;
+    }
+
+    @NonNull
+    @Override
+    public String toString() {
+        return "{"
+            + "key='" + key + '\''
+            + ", body=" + body
+            + '}';
+    }
+
+    @Override
+    public boolean equals(@Nullable Object obj) {
+        if (obj instanceof PetCollectionField) {
+            return ((PetCollectionField) obj).getKey().equals(this.getKey()) &&
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['&&' should be on a new line.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5479466)